### ISSUE TITLE
[QUESTION] HW6: Last code block of HW does not actually print saved csv

### ISSUE THREAD

**Issue by blostein** [betteridiot/b575f19#25]
# Instructions
* The more information/context you provide, the better we (the instructors and class) can collectively assist you.

* Please fill out all of the relevant sections below. Incomplete questions will not be addressed by the instruction team. 

* Questions asked ***without*** showing an attempt to answer individually will also not be addressed.

* It is bad etiquette to ask things such as "I don't know how to do this, can you tell me how?" or "I can't get this code to work, what is wrong?" if you do not provide the context and/or screenshots necessary to 
help. 

---
# Question (continued)
Describe the context or additional details that may be necessary to resolve this question.
Hi, 
In HW 6, the last code block is 
```python
# Paste your code in this code block (if needed)
variants_df = ("SELvariants.csv")
variants_df
```
which therefore prints 
"SELvariants.csv"

Is it ok to change this to 
```python
# Paste your code in this code block (if needed)
variants_df = pd.read_csv("SELvariants.csv")
variants_df
```
so that it prints out the contents of the SELvariants.csv file, even though the instructions say not to change that code block? Otherwise I do not see how the last code block will print our results. 
---
# What do you have?
Use this space to describe what resources/functions/code you have that you feel may be relevant to answering this question. 

**Additionally**, if this is a coding question, describe what **input** this problem section of code requires. By "input", we mean what `type` of data it is.

```python
# Paste your code in this code block (if needed)

```

---
# What do you need?
Use this space to describe what resources/functions/code/data you believe is required to determine if this question has been solved.

**Additionally**, if this is a coding question, describe what **output** this problem section of code requires. By "output", we mean what `type` of data it is.

```python
# Paste your code in this code block (if needed)

```

---
# What have you done?
Use this space to describe what approaches you have done in your attempts to solve this question yourself. This is the **most important* part of your question.

## If this is just a general question
Describe what your *hypothesis* of what the answer is here

## If this is a coding question
We require you to post your code, error messages, and output of your code here.

### User-written code
```python
# Paste your code here

```

### Observed error messages
```python-traceback
# Paste the observed error messages here

```

### Observed output of code (other than error messages)
```python
# Paste the output of your coding attempts here

```

---
# Checklist
Replace the space in between the brackets with an X
* [ ] Identified what you already have available to address this question
* [ ] Identified what you need to reach an acceptable answer
* [ ] Shown what you have done up to this point
    * [ ] A hypothesis if this is a general question
    * [ ] Your code, error messages, and output if this is a coding question

**Comment by mitreac** [comment]
Yes. The `pd.read_csv` is missing it should be as you mentioned:
```
variants_df = pd.read_csv("SELvariants.csv")
variants_df
```
